src/index.html

```diff
@@ -9,9 +9,9 @@
 <body>
     <section class="container-fulid">
         <div class="row justify-content-center m-0 mt-5">
-            <div class="col-5 p-0 task-container">
-                <div class="d-flex align-items-center py-3 px-3 border-bottom border-1 border-secondary-subtle">
-                    <h1 class="text-warning fs-4 m-0">Today's To Do</h1>
+            <div class="col-10 col-md-6 p-0 task-container rounded-3">
+                <div class="d-flex align-items-center p-3 border-bottom border-1 border-secondary-subtle">
+                    <h1 class="text-dark fs-3 m-0">Daily Tasks</h1>
                     <i class="fa-solid fa-arrows-rotate ms-auto"></i>
                 </div>
                 <div class="d-flex align-items-center py-3 px-3 border-bottom border-1 border-secondary-subtle">
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add a feature to your application that reset the form after adding a new task. Right now, after a user has added a task the same text remains in the input field. To help you, I suggest you refer this **[article](https://www.w3schools.com/jsref/met_form_reset.asp)**